ToolsQA/Tests/Cridentials.cs

```diff
@@ -0,0 +1,17 @@
+﻿using ToolsQA.Pages;
+
+namespace ToolsQA.Tests
+{
+    class Cridentials
+    {
+        public string _username { get; set; }
+        public string _password { get; set; }
+
+        public void Login(LoginWikiPage loginWikiPage)
```

Comment:
You're mixing responsibilities - one class should have one responsibility, so, in this class, responsibility is to store user data and do some operations on them (e.g. creating full name based on first and last). In objects like that, you cannot manipulate the page.